tests/framework/console/RequestTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+use yii\console\Request;
+use yiiunit\TestCase;
+
+/**
+ * @group console
+ */
+class RequestTest extends TestCase
+{
+    public function testResolve()
```

Comment:
the use of dataProvider would cleanup your test and provide consistency throughout the test environment